src/components/Dashboard/MonthlyGoal/MonthlyGoal.tsx

```diff
@@ -71,27 +72,37 @@ const MonthlyGoal = ({
   const { t } = useTranslation();
   const { classes } = useStyles();
   const locale = useLocale();
-  const [showHealthIndicator, setShowHealthIndicator] = useState(false);
-  const [usingMachineCalculatedGoal, setUsingMachineCalculatedGoal] =
-    useState(false);
+
+  const { data, loading: healthIndicatorLoading } = useHealthIndicatorQuery({
+    variables: {
+      accountListId,
+    },
+  });
+
+  const latestHealthIndicatorData = useMemo(
+    () => data?.healthIndicatorData.at(-1),
+    [data],
+  );
+  const showHealthIndicator = !!data?.healthIndicatorData.length;
+  const machineCalculatedGoal =
+    latestHealthIndicatorData?.machineCalculatedGoal ?? null;
+  const monthlyGoal = goal || machineCalculatedGoal || 0;
+  const receivedPercentage = received / monthlyGoal;
+  const pledgedPercentage = pledged / monthlyGoal;
+  const belowGoal = monthlyGoal - pledged;
+  const belowGoalPercentage = belowGoal / monthlyGoal;
 
   const toolTipText = useMemo(() => {
-    const formattedGoal = currencyFormat(goal, currencyCode, locale);
-    return usingMachineCalculatedGoal
+    return machineCalculatedGoal
```

Comment:
This makes the tooltip say that the user has a machine-calculated goal even if they have a staff-entered goal. The condition needs to be based on whether a staff-entered goal is available.

Reply:
Good catch